src/sentry/db/postgres/lookups/array_element_contains.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+from django.contrib.postgres.fields import ArrayField
+from django.db.models import Lookup
+
+__all__ = ["ArrayElementContains"]
+
+
+class ArrayElementContains(Lookup):
+    lookup_name = "element_contains"
+
+    def as_sql(self, compiler: Any, connection: Any) -> Any:
```

Comment:
please fill out the actual types here -- you can find the base class in `.venv/lib/python3.XX/site-packages/django-stubs/...`

typing things as `(Any) -> Any` makes it much harder to fix later since they look "typed" but really aren't

leaving them untyped results in the same typing